app/build.gradle

```diff
@@ -200,6 +200,18 @@ android {
         }
     }
 
+    if (useStaticVersionCode.toBoolean()) {
+        println("Build is being created with a static version code")
```

Comment:
I think we already print the version code somewhere else?

Reply:
Hmm, I only saw the generated version code being printed. This is kind of a heads-up that the build is based on a static version code. I don't if I should print the version code though.